csm/core/services/information.py

```diff
@@ -21,6 +21,10 @@
 class InformationService(ApplicationService):
     """Version Comptibility Validation service class."""
 
+    def __init__(self, plugin):
+        self._query_deployment_plugin = plugin
+        self.resource_id_key = {"cluster": "id"}
```

Comment:
This is not a key. It is a dict. Either the change the variable name or its value. Also, this should in in const

Reply:
removed dict as we have fixed 'id' as primary key